client/src/pages/component/navbar/navbar.js

```diff
@@ -1,35 +1,182 @@
+							fill="#2693F9"
+						/>
+					</svg>
+				</div>
+			</div>
 
-<nav className="navbar navbar-light bg-light justify-content-between">
-					<i className="navbar brand">Home page</i>
-
-					<div className="form-inline">
-						<button
-							className="btn btn-outline-success my-2 my-sm-0"
-							id="laptop-home-bt"
-						>
-							<a>
-								<Link to="/create-laptop-requests"> Register for a laptop</Link>
-							</a>
-						</button>
-						<button
-							id="Donator-home-bt"
-							className="btn btn-outline-success my-2 my-sm-0"
+			<nav className="navbar navbar-light  justify-content-centre">
+				<button id="Page-home-bt">
+					<a>
```

Comment:
Is there any reason to use `button` and `a` tags? You have `<Link></Link>` component which is already a link (`a`)

Reply:
i will try fix it,  thank you